bindings/nim/kzg_abi.nim

```diff
@@ -5,14 +5,14 @@
 import
   std/[strformat, strutils]
 
-from os import DirSep
+from os import DirSep, AltSep
 
 const
   # FIELD_ELEMENTS_PER_BLOB is overrideable from
   # compiler switch -d: or --define:
   FIELD_ELEMENTS_PER_BLOB* {.intdefine.} = 4096
   # kzgPath: c-kzg-4844 project path, removing 3 last elem
-  kzgPath  = currentSourcePath.rsplit(DirSep, 3)[0] & "/"
+  kzgPath  = currentSourcePath.rsplit({DirSep, AltSep}, 3)[0] & "/"
```

Comment:
I believe the `"/"` appended at the end is platform specific, right? Should it be replaced with `DirSep`, or in the case of cross-compiling for Windows `AltSep`?

Reply:
Both backward and forward slash will works on Windows, combination of both also works. The literal "/" will works for cross-compiling for Windows.
If we use `DirSep` or `AltSep` it will become a problem because Nim will translate it into host OS separator, not target  OS separator. IIRC there is a lengthy discussion and various reasons came up why adding cross compilation features to Nim compiler(especially about path handling) is difficult.

Reply:
No, the current solution is best because it works correctly on both Linux and Windows, both locally and when cross-compiling. We rely on the fact that Windows supports `/` as a separator.

Reply:
Okay gotcha, thanks guys.